### DISCUSSION TITLE
Dialog buttons outside box

### DISCUSSION THREAD

**Discussion by peterpowaaa** [nathanhoad/godot_dialogue_manager#426] [General]
Hello everyone, I am following the tutorial videos but my dialogue buttons are outside the box...
Please help!
<img width="815" alt="buttons" src="https://github.com/nathanhoad/godot_dialogue_manager/assets/146501907/cf2fe20b-51ce-49d2-97ae-1c57c858c435">

**Comment by peterpowaaa** [comment]
Nevermind, solved it! I went into example_balloon.tscn and placed the dialogue & responses in a VBoxContainer.

**Comment by nathanhoad** [comment]
Just to clarify - it's supposed to look like that. Any visual changes can be made to your custom balloon to better suit your game's look.
